src/utils/equalsPartial.ts

```diff
@@ -2,7 +2,13 @@ import Equals from 'fast-deep-equal/es6';
 
 function equalsPartial<T>(object: T, expected: Partial<T>): boolean {
     const entries = Object.entries(expected) as [keyof T, unknown][];
-    return entries.every(([key, value]) => Equals(object[key], value));
+    return entries.every(([key, value]) => {
+        const objectValue = object[key];
+        if (typeof objectValue === 'string' && typeof value === 'string') {
```

Comment:
Why was this change needed?

Reply:
On my system, the reported information was in capitals so I also changed the equalsPartial method to be case-insensitive.

Reply:
Then I would propose to add an extra argument here: `caseInsensitive`